components/file-input/src/file-input-field-with-list/__tests__/file-input-field-with-list.test.js

```diff
@@ -0,0 +1,32 @@
+import { configure, render, fireEvent, screen } from '@testing-library/react'
+import React from 'react'
+import { FileInputFieldWithList } from '../file-input-field-with-list.js'
+
+configure({ testIdAttribute: 'data-test' })
```

Comment:
I believe this isn't needed? We have configured that already in `./jest.testing-library.config.js`.